src/config/eventBus.js

```diff
@@ -0,0 +1,2 @@
+import Vue from 'vue'
+export default new Vue()
```

Comment:
@davidjumani what's the purpose of this, or you exploring some idea?

Reply:
Used as an event bus to transmit a refresh event between the deployvm and autogen views since it's not a popup form and can't provide / inject the parentFetchData method